src/main/scala/io/iohk/ethereum/blockchain/sync/FastSyncController.scala

```diff
@@ -56,6 +65,9 @@ class FastSyncController(
         log.warning("Cannot start fast sync, not enough peers to download from. Scheduling retry in {}", startRetryInterval)
         scheduleStartFastSync(startRetryInterval)
       }
+    case StartRegularSync =>
+      context become (handlePeerUpdates orElse regularSync())
+      self ! StartRegularSync
```

Comment:
can we just call a method here (`askForHeaders`?) ? It looks a little bit strange to resume the same message here